test/integration/externalresources/testdata/rabbitmq-server.yaml

```diff
@@ -19,24 +19,20 @@ spec:
           name: rabbitmq
           livenessProbe:
             exec:
-              command:
-                - rabbitmqctl
-                - status
-            failureThreshold: 6
-            initialDelaySeconds: 10
-            periodSeconds: 10
-            successThreshold: 1
-            timeoutSeconds: 5
+              command: ["rabbitmq-diagnostics", "status"]
+            initialDelaySeconds: 60
```

Comment:
I can see the example manifest you linked to in the description also has 60s here, but have you tried with a lower value?  This adds a lot of waiting to the `deep` suite. Or perhaps trying with a stage-1 check instead?